dataactbroker/handlers/accountHandler.py

```diff
@@ -387,6 +387,10 @@ def updateUser(self, system_email):
         if requestDict.exists("is_active"):
             is_active = bool(requestDict.getValue("is_active"))
             if not self.isUserActive(user) and is_active:
+                # Reset password count to 0
+                self.resetPasswordCount(user)
+                # Reset last login date so the account isn't expired
+                self.interfaces.userDb.updateLastLogin(user, unlock_user=True)
```

Comment:
Ok. I agree with @jworcestBAH that it would better not to blow away the last login info as a way trick the password reset trigger. Unfortunately, I think the extra work to implement this correctly is a function of our choosing to custom-code our own user and login handling.